tests/fxl-spread_auto/OPS/package.opf

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-11-24</dc:date>
+    <dc:description>When rendition:spread is "auto", the reading system can choose to display spine items in a synthetic spread.</dc:description>
+    <dc:identifier id="pub-id">fxl-spread_auto</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>rendition:spread auto</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#def-spread-auto</meta>
+    <meta property="dcterms:modified">2021-11-24T00:00:00Z</meta>
+    <meta property="rendition:layout">pre-paginated</meta>
+    <meta property="rendition:spread">auto</meta>
+  </metadata>
+  <manifest>
+    <item id="page_001" href="page_001.xhtml" media-type="application/xhtml+xml" />
+    <item id="page_002" href="page_002.xhtml" media-type="application/xhtml+xml" />
+    <item id="page_003" href="page_003.xhtml" media-type="application/xhtml+xml" />
+    <item id="page_004" href="page_004.xhtml" media-type="application/xhtml+xml" />
+    <item id="style"    href="fixed.css" media-type="text/css" />
+    <item id="stix-font" href="fonts/STIXTwoText-Regular.otf" media-type="font/otf" />
+    <item id="murmure-font" href="fonts/le-murmure.otf" media-type="font/otf" />
+    <item id="nav" properties="nav" href="nav.xhtml" media-type="application/xhtml+xml" />
+  </manifest>
```

Comment:
Similar comment as above: this test seems to be related to a _MAY_ statement. If so, the following metadata item should be added:

> `<meta property="belongs-to-collection">may</meta>`

(And thanks to @dlazin to make me implement this feature for _MAY_ tests, too! 😀)